### PR TITLE
VPC config updates

### PR DESCRIPTION
**Pull request by narkarum**

### CODE REVIEW

**Comment by narkarum** [comment]
https://github.ibm.com/alchemy-containers/armada-storage/issues/832

**Comment by arahamad** [comment]
@narkarum , Can you please run the `make fmt` for this code looks some issue.

also following will be default if not passed via configuration right?
var maxRetryAttempt = 5
var maxRetryGap = 30

**Comment by narkarum** [comment]
We are setting them in code...if they were not set.

**Comment by narkarum** [comment]
Fixed the issues.

**Comment by narkarum** [comment]
@arahamad

File: volume-providers/vpc/vpcclient/client/client.go [hunk 1]
```diff
@@ -47,12 +46,19 @@ type client struct {
 }
 
 // New creates a new instance of a SessionClient
-func New(ctx context.Context, baseURL string, httpClient *http.Client, contextID string) SessionClient {
+func New(ctx context.Context, baseURL string, httpClient *http.Client, contextID string, APIVersion string) SessionClient {
+	// Default API version
+	BackendAPIVersion := models.APIVersion
```
[the remark this answers is not in the document]
Comment (narkarum) [reply volume-providers/vpc/vpcclient/client/client.go]: DONE

**Pull request merged**